### DISCUSSION TITLE
MFFrameCapture shows picture upside down

### DISCUSSION THREAD

**Discussion by SDiestelmann** [FactoryXCode/MfPack#32] [Q&A] [answered]
I tastet the Program on my VMWare VM with the "virtual" camera of my my macbookpro
The Image is upside down. 

Is there an easy solution to solve the problem perhaps to rotate the image 180° ?

**Comment by FactoryXCode** [comment, marked as answer]
Hi.
This issue has been solved in sample MfCameraFrameCapture - SampleConverter.pas, 
where the image height should have a negative value for the formats MFVideoFormat_AYUV, MFVideoFormat_IYUV and MFVideoFormat_YUY2.

Also when the stride has a negative value, this means the picture is upside down See [IMFMediaType](https://learn.microsoft.com/en-us/windows/win32/medfound/mf-mt-default-stride-attribute)


Regards, Tony.
